.github/workflows/windows.yml

```diff
@@ -0,0 +1,113 @@
+
+      - name: Build MOAB
+        shell: bash -l {0}
+        run: |
+          cd ${HOME_PATH}
+          git clone --depth 1 https://bitbucket.org/fathomteam/moab -b 5.3.0
+          mkdir moab_build moab_install
+          cd moab_build
+          cmake ../moab \
+                -DENABLE_BLASLAPACK=OFF \
+                -DENABLE_FORTRAN=OFF \
+                -DENABLE_IMESH=OFF \
+                -DENABLE_TESTING=OFF \
+                -DENABLE_HDF5=ON \
+                -DBUILD_SHARED_LIBS=ON \
+                -G"Visual Studio 16 2019"  \
+                -DCMAKE_INSTALL_PREFIX=../moab_install/ \
+                -DHDF5_ROOT="${CONDA_LOC}" \
+                -DHDF5_hdf5_LIBRARY_RELEASE="${CONDA_LOC}/lib/libhdf5_hl.lib;${CONDA_LOC}/lib/libhdf5.lib;${CONDA_LOC}/lib/zlib.lib;${CONDA_LOC}/lib/libhdf5_cpp.lib" \
+                -DCMAKE_EXE_LINKER_FLAGS="/std:c++latest" \
+                -DCMAKE_MODULE_LINKER_FLAGS="/std:c++latest" \
+                -DCMAKE_SHARED_LINKER_FLAGS="/std:c++latest" \
+                -DCMAKE_STATIC_LINKER_FLAGS="" \
+                -DCMAKE_BUILD_TYPE=Release \
+                -DCMAKE_C_COMPILER="C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe" \
+                -DCMAKE_CXX_COMPILER="C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe"
+          cmake --build . --config Release
+          cmake --install . --config Release
+
+      - uses: actions/checkout@v2
+
+      - name: build DAGMC
+        shell: bash -l {0}
+        run: |
+          cd ${HOME_PATH}
+          mkdir build 
+          cd build
+          cmake ../dagmc \
+                -G"Visual Studio 16 2019" \
+                -DBUILD_EXE=ON \
+                -DBUILD_STATIC_EXE=ON \
+                -DBUILD_SHARED_LIBS=OFF \
+                -DBUILD_STATIC_LIBS=ON \
+                -DBUILD_TALLY=OFF \
+                -DBUILD_BUILD_OBB=OFF \
+                -DBUILD_UWUW=ON \
+                -DBUILD_MAKE_WATERTIGHT=ON \
+                -DBUILD_OVERLAP_CHECK=OFF \
+                -DBUILD_TESTS=ON \
+                -DMOAB_DIR=../moab_install \
+                -DHDF5_ROOT="${CONDA_LOC}" \
+                -DHDF5_hdf5_LIBRARY_RELEASE="${CONDA_LOC}/lib/libhdf5_hl.lib;${CONDA_LOC}/lib/libhdf5.lib;${CONDA_LOC}/lib/zlib.lib;${CONDA_LOC}/lib/libhdf5_cpp.lib" \
+                -DCMAKE_INSTALL_PREFIX=../dagmc_install/ \
+                -DCMAKE_EXE_LINKER_FLAGS="" \
+                -DCMAKE_MODULE_LINKER_FLAGS="" \
+                -DCMAKE_SHARED_LINKER_FLAGS="" \
+                -DCMAKE_STATIC_LINKER_FLAGS="" \
+                -DCMAKE_BUILD_TYPE=Release
+          cmake --build . --config Release
+          cmake --install . --config Release
+
+      - name: test DAGMC
+        shell: bash -l {0}
+        run: |
+          cp -r D:/a/DAGMC/moab_install/* D:/a/DAGMC/dagmc_install/
```

Comment:
Does this copy in all the MOAB libs & includes? If so, are we faking the tests? Should we be able to build against MOAB where it is installed?

Reply:
There is multiple thing there, but you have point:
- I took what I had for the plugin which build every library separately and then combines them into the plugin
- we need a default install folder (like `~/.local` on linux) and add it to the `windows` path

basically I don't think we are faking the test but sloppy/dirty :) 
I work on cleaning this up